sdk/tests/conformance2/reading/read-pixels-into-pixel-pack-buffer.html

```diff
@@ -202,19 +219,34 @@
 function testPackAlignment()
 {
     debug("");
-    debug("Verify that reading pixels to PIXEL_PACK buffer works fine with various PACK_ALIGNMENT values.");
-
-    runTestIteration(1, 1, 2);
-    runTestIteration(2, 1, 2);
-    runTestIteration(4, 1, 2);
-    runTestIteration(8, 1, 2);
-    runTestIteration(4, 2, 2);
-    runTestIteration(8, 2, 2);
-    runTestIteration(4, 3, 2);
-    runTestIteration(8, 3, 2);
-    runTestIteration(1, 0, 0);
-    runTestIteration(4, 1, 1);
-    runTestIteration(8, 1, 1);
+    debug("Verify that reading pixels to PIXEL_PACK buffer works fine with various pack alignments.");
+    runTestIteration(1, 3, 1, 0, 0, 0);
```

Comment:
Not going to block landing this, but a suggestion here: The wall of numbers here is getting a little hard to read, and in a lot of cases you're just passing 0 anyway. Maybe make the pack parameters a dictionary? So the calls could look more like this:

```
runTestIteration(1, 3, { packAlignment: 2 });
runTestIteration(3, 3, { packAlignment: 8, packRowLength: 2 });
runTestIteration(3, 3, { packAlignment: 8, packSkipPixels: 2 });
```